system-addon/lib/PlacesFeed.jsm

```diff
@@ -207,9 +215,23 @@ class PlacesFeed {
       case at.DELETE_HISTORY_URL:
         NewTabUtils.activityStreamLinks.deleteHistoryEntry(action.data);
         break;
+      case at.OPEN_NEW_WINDOW:
```

Comment:
I agree that `TopSitesFeed` was an odd place to have had this, but I'm not sure `PlacesFeed` is that much more appropriate… but I suppose that's @k88hudson's call to make!